web-api/tests/cmd/cli_overrides.auto.toml

```diff
@@ -37,6 +37,9 @@ stdout = """
   "embedding": {
     "directory": "assets",
     "token_size": 250
+  },
+  "tenants": {
+    "enable_legacy_tenant": true
```

Comment:
It's not really much of an issue, we rarely add options and the tests very easily point out what exactly need to change. The additional overhead needed to fix this isn't worth the benefit it would get us IMHO.